content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -168,19 +168,28 @@ encrypt all resources, even custom resources that are added after API server sta
 since part of the configuration would be ineffective. The `resources` list's processing order and precedence
 are determined by the order it's listed in the configuration. {{< /note >}}
 
-Opting out of encryption for specific resources while wildcard is enabled can be achieved by adding a new
-`resources` array item with the resource name, followed by the `providers` array item with the `identity` provider.
-For example, if '`*.*`' is enabled and you want to opt-out encryption for the `events` resource, add a new item
-to the `resources` array with `events` as the resource name, followed by the providers array item with `identity`.
-The new item should look like this:
+If you have a wildcard covering resources and want to opt out of at-rest encryption for a particular kind
+of resources, you achieve that by adding a separate `resources` array item with name of the resource that
+you want to exempt, followed by a `providers` array item where you specify the `identity` provider.
+
+For example, if '`*.*`' is enabled and you want to opt-out encryption for Events and ConfigMaps, add a
+new earlier item to the `resources`, followed by the providers array item with `identity` as the
```

Comment:
A new earlier item? As in, it must come before the wildcard entry? If yes, make it clear in the previous paragraph. The note before that paragraph mentions the order but it doesn't say whether an earlier entry takes _priority_ over later entries or the other way around